### ISSUE TITLE
[Nitpick] Add copy button to hover messages: Accessibility Improvements

### ISSUE THREAD

**Issue by pwang347** [microsoft/vscode#275330]
Testing #274923

For accessibility, if you're tabbed onto the button it should eventually show the tooltip. Compare with the copy button in the chat panel.
I'm also noticing that there is no screenreader announcement when you actually press Enter to copy something (the same issue exists in the chat panel). Ideally, there should be some feedback for blind users that something was copied.
